tests/end_to_end/test_target_snowflake.py

```diff
@@ -8,6 +14,7 @@
 DIR = os.path.dirname(__file__)
 TAP_MARIADB_ID = 'mariadb_to_sf'
 TAP_POSTGRES_ID = 'postgres_to_sf'
+TAP_MONGODB_ID = 'tap_mongo_to_sf'
```

Comment:
could we make this `mongo_to_pg` to keep with the convention

Reply:
fixed in [c88330af5082a89b11a83ef0ab6984dbadd86afb](https://github.com/transferwise/pipelinewise/pull/434/commits/c88330af5082a89b11a83ef0ab6984dbadd86afb).